docs/sphinx/conf.py.in

```diff
@@ -0,0 +1,346 @@
+copyright = u'2015-%d, %s ' % (now.year, author)
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.  Set on command-line or via ant build to real version.
+#
+version = 'UNKNOWN'
+release = 'UNKNOWN'
+
+
+# The language for content autogenerated by Sphinx. Refer to documentation
+# for a list of supported languages.
+#language = None
+
+# There are two options for replacing |today|: either, you set today to some
+# non-false value, then it is used:
+#today = ''
+# Else, today_fmt is used as the format for a strftime call.
+#today_fmt = '%B %d, %Y'
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+exclude_patterns = ['_build', 'CMakeLists.txt']
+
+# The reST default role (used for this markup: `text`) to use for all documents.
+#default_role = None
+
+# If true, '()' will be appended to :func: etc. cross-reference text.
+#add_function_parentheses = True
+
+# If true, the current module name will be prepended to all description
+# unit titles (such as .. function::).
+#add_module_names = True
+
+# If true, sectionauthor and moduleauthor directives will be shown in the
+# output. They are ignored by default.
+#show_authors = False
+
+# The name of the Pygments (syntax highlighting) style to use.
+pygments_style = 'sphinx'
+
+# A list of ignored prefixes for module index sorting.
+#modindex_common_prefix = []
+
+# Variables used to define Github extlinks
+source_branch = 'develop'
+if (ext_source_branch is not None and len(ext_source_branch) > 0 and
+    ext_source_branch != "@%s@" % ('sphinx_source_branch')):
+    source_branch = ext_source_branch
+source_user = 'ome'
+if (ext_source_user is not None and len(ext_source_user) > 0 and
+    ext_source_user != "@%s@" % ('sphinx_source_user}')):
+    source_user = ext_source_user
+
+github_root = 'https://github.com/'
+ome_files_github_root = github_root + source_user + '/files/'
+ome_files_github_tree = ome_files_github_root + 'tree/' + source_branch + '/'
+ome_files_github_blob = ome_files_github_root + 'blob/' + source_branch + '/'
+gpl_formats = ome_files_github_blob + 'components/formats-gpl/src/loci/formats/'
+bsd_formats = ome_files_github_blob + 'components/formats-bsd/src/loci/formats/'
+ome_files_cpp = ome_files_github_blob + 'cpp/'
+
+# Variables used to define Jenkins extlinks
+jenkins_root = 'http://ci.openmicroscopy.org'
+jenkins_job_root = jenkins_root + '/job'
+jenkins_view_root = jenkins_root + '/view'
+
+# Variables used to define other extlinks
+cvs_root = 'http://cvs.openmicroscopy.org.uk'
+trac_root = 'https://trac.openmicroscopy.org/ome'
+oo_root = 'http://www.openmicroscopy.org'
+oo_site_root = oo_root + '/site'
+lists_root = 'http://lists.openmicroscopy.org.uk'
+downloads_root = 'http://downloads.openmicroscopy.org'
+omerodoc_uri = oo_site_root + '/support/omero5.1'
```

Comment:
this should probably just be /omero/ so it auto-redirects to the latest version. If that's going to cause issues, it should at least be 5.2